R/prepare-admin-panel.R

```diff
@@ -759,8 +759,10 @@ prepare_admin_panel_components <- function(
 
   output$sessions_general <- timevis::renderTimevis({
     timevis::timevis(sessions_data(), options = list(
-      start = as.POSIXct(sprintf("%s 00:00:00", Sys.Date())),
-      end = as.POSIXct(Sys.time()),
+      start = as.POSIXct(
+        sprintf("%s 00:00:00", lubridate::today(tzone = "UTC"))
+      ),
+      end = as.POSIXct(lubridate::now(tzone = "UTC")),
```

Comment:
This can be simplified by just using `today()`

```
as.POSIXct(
    sprintf("%s 00:00:00", lubridate::today(tzone = "UTC")),
    tz = "UTC"
) == lubridate::today(tzone = "UTC")
```

![image](https://user-images.githubusercontent.com/211358/231991399-4457b1b5-b13a-487f-8a47-8557d2ee5970.png)